src/db-access/expenses.ts

```diff
@@ -86,3 +87,78 @@ export async function selectExpenses(
 		})
 		.then(handleDatabaseFullfillment, handleDatabaseRejection);
 }
+
+export async function getSumUnprocessed(): Promise<DatabaseResult<number>> {
+	return database
+		.transaction(async (tx) => {
+			return (await tx.select().from(expensesTable).where(isNull(expensesTable.handlingDate))).reduce(
+				(accumulator, currentValue) => {
+					const moneyParseResult = z.coerce.number().positive().safeParse(currentValue.moneyAmount);
+					if (moneyParseResult.success){
+						return accumulator + moneyParseResult.data;
+					}else{
+						throw ormError("Invalid money number from database.");
+					}
+				},
+				0
+			)
+		})
+		.then(handleDatabaseFullfillment, handleDatabaseRejection);
+}
+
+export async function getSumAccepted(): Promise<DatabaseResult<number>> {
+	return database
+		.transaction(async (tx) => {
+			return (await tx.select().from(expensesTable).where(and(expensesTable.isAccepted, isNotNull(expensesTable.handlingDate)))).reduce(
+				(accumulator, currentValue) => {
+					const moneyParseResult = z.coerce.number().positive().safeParse(currentValue.moneyAmount);
+					if (moneyParseResult.success){
+						return accumulator + moneyParseResult.data;
+					}else{
+						throw ormError("Invalid money number from database.");
+					}
+				},
+				0
+			)
+		})
+		.then(handleDatabaseFullfillment, handleDatabaseRejection);
+}
+
+export async function getSumRejected(): Promise<DatabaseResult<number>> {
+	return database
+		.transaction(async (tx) => {
+			return (await tx.select().from(expensesTable).where(and(not(expensesTable.isAccepted), (isNotNull(expensesTable.handlingDate))))).reduce(
+				(accumulator, currentValue) => {
+					const moneyParseResult = z.coerce.number().positive().safeParse(currentValue.moneyAmount);
+					if (moneyParseResult.success){
+						return accumulator + moneyParseResult.data;
+					}else{
+						throw ormError("Invalid money number from database.");
+					}
+				},
+				0
+			)
+		})
+		.then(handleDatabaseFullfillment, handleDatabaseRejection);
+}
+
+export async function getAgeragePaybackTime(): Promise<DatabaseResult<number>> {
+	return database
+		.transaction(async (tx) => {
+			const result = (await tx.select().from(expensesTable).where(isNotNull(expensesTable.handlingDate)));
+
+			if (result.length === 0){
+				return 0;
+			}
+
+			const totalMilliseconds = result.reduce(
+				(accumulator, currentValue) => {
+					return accumulator + ((currentValue.handlingDate as Date).getTime() - currentValue.submitDate.getTime());
```

Comment:
```suggestion
					return accumulator + ((currentValue.handlingDate as Date).getTime() - currentValue.submitDate.getTime());
```
Make this type assertion into its own line, and add a comment explaining why it is ok to use it in this case.